gulpfile.js

```diff
@@ -47,21 +47,32 @@ function clean() {
   return del(["./vendor/"]);
 }
 
+function module_path(module) {
+  // Yarn 2 Pnp
+  if (process.versions.pnp) {
+    return (require('path')).dirname(require.resolve(`${module}/package.json`));
+  }
+  // Yarn 1 (classic)
+  if (!process.versions.pnp) {
+    return `./node_modules/${module}`;
+  }
+}
```

Comment:
This should use `require.resolve` no matter what the user is running under, hardcoding `node_modules` paths is asking for issues, if for example this was used in a monorepo the path would (probably) not exist

Reply:
I agree. I will update PR once I have some free time.